### DISCUSSION TITLE
Floating window error

### DISCUSSION THREAD

**Discussion by ghost** [w0rp/ale#3702] [Q&A - Ask for help with problems] [answered]
vim version ：Vim82

Configuration information:
 let g:ale_floating_preview = 1
 let g:ale_hover_to_floating_preview = 1
 let g:ale_detail_to_floating_preview = 1
 let g:ale_use_global_executables = 1
 let g:float_preview#docked = 0
 let g:ale_cursor_detail = 1
 let g:ale_hover_cursor = 1
 let g:ale_close_preview_on_insert = 1

I want to use floating window in vim82, but it doesn't work，why ？
![Screenshot_20210414_212209](https://user-images.githubusercontent.com/61009000/114717350-8bf1bd00-9d67-11eb-8258-981cdf85f564.jpg)

**Comment by hsanson** [comment, marked as answer]
Floating windows is a Neovim only feature.
